.buildkite/shared/pants.toml

```diff
@@ -0,0 +1,57 @@
+[GLOBAL]
+pants_version = "2.7.1"
+backend_packages = [
+    "pants.backend.shell",
+    "pants.backend.shell.lint.shellcheck",
+    "pants.backend.shell.lint.shfmt"
+]
+
+pants_ignore = [
+    "!.buildkite/"
```

Comment:
Is there a reason to set this? tests and hooks both contain bash code that could benefit from shellcheck and shfmt

Reply:
This particular one is from the git subtree, so it doesn't really "count". However, it's in the top-level `pants.toml` file, and it probably shouldn't be. I think it came in via copy/paste initially, when it really shouldn't have.

Thanks for catching this; probably need to remove it from our other plugins, too